models/orders_stg/orders_stg.sql

```diff
@@ -9,3 +9,5 @@ SELECT
 	Status
 FROM
 	{{ ref('orders') }}
+WHERE
+	Status != 'cancelled'
```

Comment:
How about also dealing with pending orders
```suggestion
	Status NOT IN ('cancelled', 'pending')
```